<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoToolsActivating</class>
 <widget class="QWidget" name="DemoToolsActivating">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <item>
      <widget class="PwGisWidget" name="pwgiswidget" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>170</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="mapLayout">
           <item>
            <widget class="QPushButton" name="btnOpenMap">
             <property name="text">
              <string>Open Map</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCloseMap">
             <property name="text">
              <string>Close Map</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="atlasLayout">
           <item>
            <widget class="QPushButton" name="btnOpenAtlas">
             <property name="text">
              <string>Open Atlas</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCloseAtlas">
             <property name="text">
              <string>Close Atlas</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="profileLayout">
           <item>
            <widget class="QPushButton" name="btnRegisterProfile">
             <property name="text">
              <string>Register profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnUnregisterProfile">
             <property name="text">
              <string>Unregister profile</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="bottomMargin">
        <number>300</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>344</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>map tools:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="actiavateTool">
         <property name="text">
          <string>Activate NavigateTool</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="actiavateTool_2">
         <property name="text">
          <string>Activate DrawCircle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="actiavateTool_6">
         <property name="text">
          <string>Activate Draw Rectangle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="actiavateTool_5">
         <property name="text">
          <string>Activate Ruler</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="actiavateTool_4">
         <property name="text">
          <string>Activate Curvometr</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="actiavateTool_3">
         <property name="text">
          <string>Activate Draw Point</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="actiavateTool_7">
         <property name="text">
          <string>Activate Draw Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="actiavateTool_8">
         <property name="text">
          <string>Change Geometry</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>344</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>map buttons:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTool_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>NavigateToolButton</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTool_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>DrawCircleButton</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTool_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Draw Rectangle Button</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTool">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ruler Button</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTool_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Curvometr Button</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTool_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Draw Point Button</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTool_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Draw Path Button</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTool_9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Change Geometry Button</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
